tool.ts

```diff
@@ -485,7 +486,10 @@ function _createExtractFolder(dest?: string): string {
         dest = path.join(_getAgentTemp(), uuidV4());
     }
 
-    tl.mkdirP(dest);
+    if (!tl.exist(dest)) {
```

Comment:
mkdirp checks whether the destination exists, and if exists validates its a directory. let mkdirp do the work.